README.md

```diff
@@ -1874,6 +1874,247 @@ custom_keys:
+  '7':
+    tap_action:
+      action: call-service
+      service: remote.send_command
+      target:
+        entity_id: remote.rm4_pro
+      data:
+        device: Office TV Led
+        command: orange-light
+    style:
+      height: 35px;
+      width: 35px;
+      background-color: lightsalmon;
+      border-radius: 50%;
+  '8':
+    tap_action:
+      action: call-service
+      service: remote.send_command
+      target:
+        entity_id: remote.rm4_pro
+      data:
+        device: Office TV Led
+        command: cyan
+    style:
+      height: 35px;
+      width: 35px;
+      background-color: cyan;
+      border-radius: 50%;
+  '9':
+    tap_action:
+      action: call-service
+      service: remote.send_command
+      target:
+        entity_id: remote.rm4_pro
+      data:
+        device: Office TV Led
+        command: blue
+    style:
+      height: 35px;
+      width: 35px;
+      background-color: blue;
+      border-radius: 50%;
+  '10':
+    tap_action:
+      action: call-service
+      service: remote.send_command
+      target:
+        entity_id: remote.rm4_pro
+      data:
+        device: Office TV Led
+        command: pink
+    style:
+      height: 35px;
+      width: 35px;
+      background-color: magenta;
+      border-radius: 50%;
+  '11':
+    tap_action:
+      action: call-service
+      service: remote.send_command
+      target:
+        entity_id: remote.rm4_pro
+      data:
+        device: Office TV Led
+        command: green_light
+    style:
+      height: 35px;
+      width: 35px;
+      background-color: mediumseagreen;
+      border-radius: 50%;
+  '12':
+    tap_action:
+      action: call-service
+      service: remote.send_command
+      target:
+        entity_id: remote.rm4_pro
+      data:
+        device: Office TV Led
+        command: white
+    style:
+      height: 35px;
+      width: 35px;
+      background-color: white;
+      border-radius: 50%;
+  power:
+    tap_action:
+      action: call-service
+      service: remote.send_command
+      target:
+        entity_id: remote.rm4_pro
+      data:
+        device: Office TV Led
+        command: 'on'
+    style:
+      color: green
+  poweroff:
+    icon: mdi:power
+    tap_action:
+      action: call-service
+      service: remote.send_command
+      target:
+        entity_id: remote.rm4_pro
+      data:
+        device: Office TV Led
+        command: 'off'
+    style:
+      color: red
+  up:
+    tap_action:
+      action: call-service
+      service: remote.send_command
+      target:
+        entity_id: remote.rm4_pro
+      data:
+        device: Office TV Led
+        command: brightness+
+  down:
+    tap_action:
+      action: call-service
+      service: remote.send_command
+      target:
+        entity_id: remote.rm4_pro
+      data:
+        device: Office TV Led
+        command: brightness-
+```
+<img src="https://github.com/SylTi/android-tv-card/blob/main/assets/rgb.png" alt="rgb remote example" width="500"/>
```

Comment:
Gotta update this to point to my repo instead (replace SylTi with Nerwyn)

Reply:
Done!